autometa/binning/recursive_dbscan.py

```diff
@@ -703,8 +703,8 @@ def main():
         "--reverse-ranks",
         action="store_true",
         default=False,
-        help="Reverse canonical-rank taxonomy splitting"
-        " from superkingdom -> species to species -> superkingdom",
+        help="Reverse order at which to split taxonomy by canonical-rank."
+        " Contigs will be split in order of species, genus, family, order, class, phylum, superkingdom.",
     )
```

Comment:
This is still not completely clear. I think you will find users asking you exactly what order is --reverse-ranks and which is not.

```suggestion
        help="Reverse order at which to split taxonomy by canonical-rank."
        "When --reverse-ranks given, contigs will be split in order of species, genus, family, order, class, phylum, superkingdom.",
    )
```